python/ray/tests/test_actor_failures.py

```diff
@@ -23,6 +21,31 @@
 SIGKILL = signal.SIGKILL if sys.platform != "win32" else signal.SIGTERM
 
 
+def init_pubsub():
```

Comment:
Why don't we move these to test utils instead?

Reply:
fixed